docs/on-prem.rst

```diff
@@ -883,13 +883,16 @@ The security in place between |sr| and the end clients, e.g. ``appSA``, is as fo
            --producer-property ssl.keystore.password=confluent \
            --producer-property ssl.key.password=confluent
 
-   The error should resemble:
+   After starting the console producer, it will wait for input. Enter a few characters and press enter.
+   It should result in a failure with an error message that resembles:
 
    .. sourcecode:: bash
 
       ERROR Error when sending message to topic users with key: null, value: 5 bytes with error: (org.apache.kafka.clients.producer.internals.ErrorLoggingCallback)
       org.apache.kafka.common.InvalidRecordException: This record has failed the validation on broker and hence be rejected.
```

Comment:
I understand this message is auto-generated, but the wording is unfortunate - wonder if eng could change it? 
This record has failed the validation on the broker and has been rejected.

Reply:
@JoyceFee1 a product jira already exists for that https://confluentinc.atlassian.net/browse/DGS-247